src/common.ts

```diff
@@ -0,0 +1,26 @@
+import BN from 'bn.js';
+
+export interface Options {
+  contractAddress?: string;
+}
+
+export type BalanceMap = {
+  [tokenAddress: string]: BN;
+}
+
+export type AddressBalanceMap = {
+  [address: string]: BalanceMap;
```

Comment:
I wonder if its better to in-line BalanceMap into AddressBalanceMap for clarity.

Reply:
You can actually access the internal type by doing `AddressBalanceMap[string]`. But I agree this should be its seperate type of you're using it somewhere else